frontend/apps/reader/modules/readersearch.lua

```diff
@@ -177,19 +177,21 @@ function ReaderSearch:onShowFulltextSearchInput()
         end,
     }
 
-    local checkbox_shift = math.floor((self.input_dialog.width - self.input_dialog._input_widget.width) / 2 + 0.5)
-    local check_buttons = HorizontalGroup:new{
-        HorizontalSpan:new{width = checkbox_shift},
-        VerticalGroup:new{
-            align = "left",
-            self.check_button_case,
-            self.check_button_regex,
-        },
-    }
+    if self.ui.document.provider == "crengine" then
```

Comment:
Noooo, I don't want to see any more of that ! :)
https://github.com/koreader/koreader/pull/7883#discussion_r658030644